packages/node/src/evm/verification/request-verification.ts

```diff
@@ -16,19 +16,6 @@ export function verifyDesignatedWallets<T>(
       return [[log], request];
     }
 
-    // We can't validate the wallet if there is no requester index. There should always be one
-    // at this point but just in case there isn't.
-    if (!request.requesterIndex) {
```

Comment:
Since the `requesterIndex` is now non-nullable this check seems unnecessary so I removed it.